packages/machine-extractor/src/extractMachinesFromFile.ts

```diff
@@ -1,11 +1,12 @@
+import { MachineExtractResult } from '.';
```

Comment:
this creates a cyclic dependency (`src/index` depends already on this file, `src/extractMachinesFromFile`, and now this file depends on `src/index` too). I would reach to the actual source for this import:

 ```suggestion
import { MachineExtractResult } from './MachineExtractResult';
```

Reply:
I removed the type in the latest commit as you added the default, and thanks for that 👍🏻